string id
float64 timestamp
float64[] state
